experiments/ClimaEarth/components/atmosphere/climaatmos.jl

```diff
@@ -449,7 +449,7 @@ function get_atmos_config_dict(coupler_dict::Dict, job_id::String, atmos_output_
 
     # use atmos toml if coupler toml is not defined
     atmos_toml = joinpath.(pkgdir(CA), atmos_config["toml"])
-    coupler_toml = joinpath.(pkgdir(ClimaCoupler), coupler_dict["coupler_toml"])
+    coupler_toml = atmos_config["coupler_toml"]
```

Comment:
It might be a little confusing to have these two lines together, where we have to prepend `pkgdir` for the atmos toml but we've already done it for the coupler. We could add a comment explaining why, but I don't think it matters too much either way

Reply:
I'll add a comment, thanks!